packages/apidom-ns-asyncapi-3/package.json

```
@@ -0,0 +1,54 @@
+{
+  "name": "@swagger-api/apidom-ns-asyncapi-3",
+  "version": "1.0.0-beta.0",
+  "description": "AsyncAPI 3.x.y namespace for ApiDOM.",
+  "publishConfig": {
+    "access": "public",
+    "registry": "https://registry.npmjs.org"
+  },
+  "type": "module",
+  "sideEffects": [],
+  "main": "./src/index.cjs",
+  "exports": {
+    "types": "./types/apidom-ns-asyncapi-3.d.ts",
+    "import": "./src/index.mjs",
+    "require": "./src/index.cjs"
+  },
+  "types": "./types/apidom-ns-asyncapi-3.d.ts",
+  "scripts": {
+    "build": "npm run clean && run-p --max-parallel ${CPU_CORES:-2} typescript:declaration build:es build:cjs build:umd:browser",
+    "build:es": "cross-env BABEL_ENV=es babel src --out-dir src --extensions '.ts' --out-file-extension '.mjs' --root-mode 'upward'",
+    "build:cjs": "cross-env BABEL_ENV=cjs babel src --out-dir src --extensions '.ts' --out-file-extension '.cjs' --root-mode 'upward'",
+    "build:umd:browser": "cross-env BABEL_ENV=browser webpack --config config/webpack/browser.config.js --progress",
+    "lint": "eslint ./",
+    "lint:fix": "eslint ./ --fix",
+    "clean": "rimraf --glob 'src/**/*.mjs' 'src/**/*.cjs' ./dist ./types",
+    "test": "NODE_ENV=test ts-mocha --project tsconfig.json --exit \"test/**/*.ts\"",
+    "typescript:declaration": "tsc -p tsconfig.declaration.json && api-extractor run -l -c ./config/api-extractor/api-extractor.json 2>&1 | shx grep -v 'Visitor_base'"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/swagger-api/apidom.git"
+  },
+  "author": "apidom team",
+  "license": "Apache-2.0",
+  "dependencies": {
+    "@babel/runtime-corejs3": "^7.26.10",
+    "@swagger-api/apidom-core": "^1.0.0-beta.51",
+    "@swagger-api/apidom-ns-asyncapi-2": "workspace:^1.0.0-beta.51",
```

We need to change this and run `npm i` in the root repo (maybe also in this namespace)

```suggestion
    "@swagger-api/apidom-ns-asyncapi-2": "^1.0.0-beta.51",
```